consensus/vbft/utils.go

```diff
@@ -356,11 +349,14 @@ func getShardConfigByShardID(lgr *ledger.Ledger, shardID common.ShardID, blkNum
 		return nil, err
 	}
 	var blkHeight uint32
+	sort.Slice(heights, func(i, j int) bool { return heights[i] < heights[j] })
```

Comment:
heights should be a increasing order, optimize these code and AddShardConsensusHeight function